apps/communications/dialer/test/marionette/lib/views/keypad/views.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+/* global module */
+var KeypadAccessor = require('./accessors');
+
+function KeypadView(client) {
+  
+  this.client = client;
+  this.accessors = new KeypadAccessor(client);
+
+}
+
+KeypadView.prototype = {
+
+  key: function(keyNumber) {
```

Comment:
Nit: This function returns an HtmlElement. For a matter of consistency, it should belong with the accessors.